content/en/contribute/development.md

```diff
@@ -97,12 +97,26 @@ Step 4
 : Make changes.
 
 Step 5
-: Compile and install:
+: Compile and install.
+
+To compile and install the standard edition:
+
+```text
+CGO_ENABLED=1 go install
+```
```

Comment:
You don't need CGO for the standard edition.

Reply:
Mindless copy/paste error. Thanks for the catch.